### PR TITLE
add timer.jquery@0.6.4 w/ git auto-update

### LINKED ISSUE #8626: [Request] Add jQuery Timer plugin
**Library name:** timer.jquery
**Git repository url:** https://github.com/walmik/timer.jquery
**npm package url(optional):** 
**License(s):**
**Official homepage:** http://jquerytimer.com/
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by dada0423**
close #8626, cc @walmik

Pull request for issue: #8626
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/walmik/timer.jquery
 * Official website (optional, not the repository):http://jquerytimer.com
 * NPM package url (optional):
 * License and its reference:
 * GitHub / Bitbucket popularity (required):
   - Count of stars:184
   - Count of watchers:25
   - Count of forks:64
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by dada0423** [comment]
@pvnr0082t  Please help me review this PR. Thank you

File: ajax/libs/timer.jquery/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "timer.jquery",
+  "filename": "timer.jquery.min.js",
+  "description": "Start/Stop/Resume/Remove a timer inside any HTML element.",
+  "author": "Walmik Deshpande",
+  "version": "0.6.4",
+  "homepage": "jquerytimer.com",
```
Comment (pvnr0082t) [inline ajax/libs/timer.jquery/package.json]: @dada0423 Is this url?

**Comment by walmik** [comment]
@dada0423 Thank you so much for this.. I always wanted the jQuery timer plugin to be on CDNjs!  Is there anything you need from me to continue with this?

**Comment by dada0423** [comment]
@pvnr0082t I've corrected it.

**Comment by pvnr0082t** [comment]
@walmik Thank you. We just notify you that we're going to host this library. We will open an issue If we need help. :smile:

**Comment by pvnr0082t** [comment]
LGTM
@cdnjs/intern2 Please help review this PR, thank you.

**Comment by kennynaoh** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
@walmik I think you can help us tell your users that they can use timer.jquery on CDNJS from now!

**Comment by walmik** [comment]
A very big thank you to @dada0423 @pvnr0082t @PeterDaveHello @kennynaoh ... sincerely!

**Comment by PeterDaveHello** [comment]
@walmik you are welcome

**Pull request merged**